pkg/dataMover/data_mover.go

```diff
@@ -101,6 +112,13 @@ func (this *DataMover) CopyToRepo(peID astrolabe.ProtectedEntityID) (astrolabe.P
 	this.RegisterOngoingUpload(peID, cancelFunc)
 
 	log.Debugf("Ready to call s3 PETM copy API for local PE")
+	if backupRepository.Name != "Not use" {
```

Comment:
can we create a constant, and use it here?, something like "not-applicable"

Reply:
If Name is "Not use" how do we avoid an NPE at line 104

Reply:
> If Name is "Not use" how do we avoid an NPE at line 104

line 104 is "updatedPE, err := this.ivdPETM.GetProtectedEntity(ctx, peID)", this will not cause an NPE.